<ui version="4.0" >
 <class>CMakeFirstConfigure</class>
 <widget class="QDialog" name="CMakeFirstConfigure" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>First Configure Setup</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Please select what build system you want CMake to generate files for.  You should select the tool that you will use to build the project.</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QComboBox" name="generators" />
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QRadioButton" name="useDefaults" >
       <property name="text" >
        <string>Use Defaults</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="compilerSetup" >
       <property name="text" >
        <string>Compiler Setup</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="crossCompilerSetup" >
       <property name="text" >
        <string>Cross Compiler Setup</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QStackedWidget" name="stackedWidget" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex" >
      <number>2</number>
     </property>
     <widget class="QWidget" name="defaultPage" >
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item row="0" column="1" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>The default compilers will be used.</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="compilerPage" >
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QGroupBox" name="groupBox_4" >
         <property name="title" >
          <string>Compilers</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_16" >
            <property name="text" >
             <string>C</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QCMakeFilePathEditor" name="CCompiler" />
          </item>
          <item row="0" column="2" >
           <widget class="QLabel" name="label_17" >
            <property name="text" >
             <string>C++</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3" >
           <widget class="QCMakeFilePathEditor" name="CXXCompiler" />
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_18" >
            <property name="text" >
             <string>Fortran</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QCMakeFilePathEditor" name="FortranCompiler" />
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>566</width>
           <height>71</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="crossCompilerPage" >
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item row="1" column="0" >
        <widget class="QStackedWidget" name="toolChainStack" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex" >
          <number>1</number>
         </property>
         <widget class="QWidget" name="page" >
          <layout class="QGridLayout" >
           <property name="leftMargin" >
            <number>9</number>
           </property>
           <property name="topMargin" >
            <number>9</number>
           </property>
           <property name="rightMargin" >
            <number>9</number>
           </property>
           <property name="bottomMargin" >
            <number>9</number>
           </property>
           <item row="0" column="1" >
            <widget class="QCMakeFilePathEditor" name="toolChainFile" />
           </item>
           <item row="1" column="1" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0" >
            <widget class="QLabel" name="label_5" >
             <property name="text" >
              <string>Tool Chain File</string>
             </property>
             <property name="wordWrap" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2" >
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <widget class="QGroupBox" name="groupBox" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title" >
              <string>System</string>
             </property>
             <layout class="QGridLayout" >
              <item row="0" column="0" >
               <layout class="QHBoxLayout" >
                <item>
                 <widget class="QLabel" name="label_6" >
                  <property name="sizePolicy" >
                   <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text" >
                   <string>Name</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="systemName" />
                </item>
                <item>
                 <widget class="QLabel" name="label_10" >
                  <property name="sizePolicy" >
                   <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text" >
                   <string>Version</string>
                  </property>
                  <property name="wordWrap" >
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="systemVersion" />
                </item>
                <item>
                 <widget class="QLabel" name="label_11" >
                  <property name="sizePolicy" >
                   <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text" >
                   <string>Processor</string>
                  </property>
                  <property name="wordWrap" >
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="systemProcessor" />
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QGroupBox" name="groupBox_3" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title" >
              <string>Compilers</string>
             </property>
             <layout class="QGridLayout" >
              <item row="0" column="0" >
               <widget class="QLabel" name="label_8" >
                <property name="text" >
                 <string>C</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1" >
               <widget class="QCMakeFilePathEditor" name="crossCCompiler" />
              </item>
              <item row="0" column="2" >
               <widget class="QLabel" name="label_7" >
                <property name="text" >
                 <string>C++</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3" >
               <widget class="QCMakeFilePathEditor" name="crossCXXCompiler" />
              </item>
              <item row="1" column="0" >
               <widget class="QLabel" name="label_15" >
                <property name="text" >
                 <string>Fortran</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1" >
               <widget class="QCMakeFilePathEditor" name="crossFortranCompiler" />
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0" >
            <widget class="QGroupBox" name="groupBox_2" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title" >
              <string>Find Program/Library/Include</string>
             </property>
             <layout class="QGridLayout" >
              <item row="0" column="0" >
               <widget class="QLabel" name="label_9" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text" >
                 <string>Root</string>
                </property>
                <property name="wordWrap" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="1" >
               <widget class="QCMakePathEditor" name="crossFindRoot" />
              </item>
              <item row="0" column="2" >
               <widget class="QLabel" name="label_12" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text" >
                 <string>Program Mode</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3" >
               <widget class="QComboBox" name="crossProgramMode" >
                <item>
                 <property name="text" >
                  <string>Find from Root then system</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>Only find from Root</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>Don't find from Root</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="0" >
               <widget class="QLabel" name="label_13" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text" >
                 <string>Library Mode</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1" >
               <widget class="QComboBox" name="crossLibraryMode" >
                <item>
                 <property name="text" >
                  <string>Find from Root then system</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>Only find from Root</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>Don't find from Root</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="2" >
               <widget class="QLabel" name="label_14" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text" >
                 <string>Include Mode</string>
                </property>
               </widget>
              </item>
              <item row="1" column="3" >
               <widget class="QComboBox" name="crossIncludeMode" >
                <item>
                 <property name="text" >
                  <string>Find from Root then system</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>Only find from Root</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>Don't find from Root</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QCheckBox" name="useToolChainFile" >
         <property name="text" >
          <string>Use ToolChain File</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="6" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCMakeFilePathEditor</class>
   <extends>QLineEdit</extends>
   <header>QCMakeWidgets.h</header>
  </customwidget>
  <customwidget>
   <class>QCMakePathEditor</class>
   <extends>QLineEdit</extends>
   <header>QCMakeWidgets.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CMakeFirstConfigure</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>227</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CMakeFirstConfigure</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>295</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
